Makefile

```diff
@@ -1,10 +1,13 @@
 test:
-	pytest cupyd/tests/ -v -ra
+	pytest cupyd/tests/ -v -ra --cov=cupyd/core

```

Comment:
we will use the pytest coverage plugin to export the coverage file without needing extra commands